clients/context-registry/package.json

```diff
@@ -36,7 +36,8 @@
     "url": "http://www.bentley.com"
   },
   "dependencies": {
-    "deep-assign": "^2.0.0"
+    "deep-assign": "^2.0.0",
+    "lodash": "^4.17.10"
```

Comment:
Maybe pull in [lodash.once](https://www.npmjs.com/package/lodash.once) instead? Need to look into how much of this we can tree-shake later

Reply:
I wasn't going to do too much refactoring of the client impl themselves as we intend to completely replace these.